index.html

```diff
@@ -893,14 +932,58 @@ <h3>Presentations</h3>
+          The verifiable credential graph contains credential metadata and other claims.
+        </p>
+        
+        <figure id="info-graph-vp-jwt">
+          <img style="margin: auto; display: block; width: 100%;" src="diagrams/vp-jwt.svg" alt="Diagram with, on the left,
+                      a box, labeled as 'JWT (Decoded)', and with three textual labels
+                      stacked vertically, namely 'Header', 'Payload', and 'Signature'.
+                      The 'Header' label is connected, with an arrow, to a separate rectangle 
+                      on the right side of the diagram containing three text 
+                      fields: 'kid: https://example.com/keys/#1234', 'alg: E384', and 'cty: vp+ld+json'. 
+                      The 'Payload' label of the left side is connected, with an arrow, to a separate rectangle, 
+                      consisting of two related graphs (stacked vertically) connected 
+                      by a an arrow labeled 'verifiableCredential'.
+                      The two graphs have each a label 'verifiable presentation graphs (serialized in JSON)' and 
+                      'verifiable credential graph (serialized in JSON)', respectively.
+                      The top graph in the rectangle has and object 'Presentation ABC' with 3 properties: 'type'
+                      of value VerifiablePresentation, 'termsOfUse' of value 'Do Not Archive'.
+                      The bottom graph includes 'Credential 123' as a subject 
+                      with 4 properties: 'type' of value ExampleAlumniCredential,
+                      'issuer' of Example University, 'validFrom' of 2010-01-01T19:23:24Z, and
+                      credentialSubject of Pat, who also has an 'alumniOf' property with value of
+                      'Example University'.
+                      Finally, the 'Signature' label on the left side is connected, with an 
+                      arrow, to a separate rectangle, containing a single text field:
+                      'XaOOh4ljklxH7L99RTVSfOl...'.">
+          <figcaption style="text-align: center;">
+            Information graphs associated with a basic verifiable presentation, using an [=enveloping proof=]
+            based on [[[VC-JOSE-COSE]]] [[?VC-JOSE-COSE]].
+          </figcaption>
+        </figure>
+
+
         <p class="note">
 It is possible to have a [=presentation=], such as a business persona, which
 draws on multiple [=credentials=] about different [=subjects=] that are
 often, but not required to be, related.
+This is achieved by using the <code>verifiableCredential</code> property to
+refer to multiple verifiable credentials. In the [=embedded proof=] case this means adding several verifiable credential
+graphs, each with its own, separate proof graph; the number of information graphs becomes then six, eight, etc.
+In the [=enveloping proof=] case the additional verifiable credential graphs are added to the same payload.
```

Comment:
I'm not sure I follow this sentence, can you elaborate?

Reply:
The `verifiableCredential` property can refer to several credentials. I.e.

```json
{
   …
   "verifiableCredential": [{
         ... Credential #1
    },{
         ... Credential #2    
    },{
         ... Credential #3   
    }]
}
```

Each credential is a separate (named) graph, and each credential graph has its own related proof graph. In the embedded proof case each credential, and its related proof graph, is added to the collection of graphs which form the complete presentation.

This is what the sentence tries to describe; clearly does not do a good job at it. Any better (but still succinct) way of expressing this?

Reply:
IMO, it would suffice to only include the first sentence here. I would leave only the text below. 

> This is achieved by using the <code>verifiableCredential</code> property to
> refer to multiple verifiable credentials.

And remove the text "In the [=embedded proof=] ... same payload."

Reply:
I wait a few days to see if someone has a different opinion, but I am happy to make this change.